clients/js/cmds/sui.ts

```diff
@@ -59,79 +107,139 @@ exports.builder = function(y: typeof yargs) {
       const rpc = argv.rpc ?? NETWORKS[network]["sui"].rpc;
       console.log("network: ", network)
       console.log("rpc: ", rpc)
-      await publishPackage(network, rpc, `${dir}/tokenbridge`);
+      await publishPackage(network, rpc, `${dir}/token_bridge`);
     })
     .command("init-wormhole", "Init wormhole core contract", (yargs) => {
       return yargs
         .option("network", network_options)
         .option("rpc", rpc_description)
-        .option("deployer", {
-          alias: "d",
-          describe: "Deployer capability object ID",
+        .option("package-id", {
+          describe: "Package/module ID",
           required: true,
-          type: "string",
+          type: "string"
         })
         .option("chain-id", {
           alias: "ci",
           describe: "Chain ID",
-          default: 22,
+          default: "22",
           required: false,
-          type: "number"
+          type: "string"
         })
         .option("governance-chain-id", {
           alias: "gci",
           describe: "Governance chain ID",
-          default: 1,
-          type: "number",
+          default: "3",
```

Comment:
can you explain why this is 3? i would expect it to be 1

Reply:
yes, perhaps